simulator/index.md

```diff
@@ -39,6 +39,11 @@ You'll need to ensure a matching version of Python is installed. If you're still
 having problems, ask for help in [`#simulator-help`][simulator-help] in
 [Discord][discord].
 
+On Windows your Python path is likely `C:\Users\<USERNAME>\AppData\Local\Programs\Python\Python37\python.exe` where `<USERNAME>` is your login.
```

Comment:
```suggestion
On Windows your Python path is likely `C:\Users\<USERNAME>\AppData\Local\Programs\Python\Python39\python.exe` where `<USERNAME>` is your login.
```

Reply:
Separately, is "login" the term which people are likely to understand? Would "username" be better?